index.html

```diff
@@ -76,6 +76,12 @@ <h1 id="title"><i class="todo">PROPOSED</i> Verifiable Credentials Working Group
 
       <p class="mission">The <strong>mission</strong> of the <a href="">Verifiable Credentials Working Group</a> is to make expressing, exchanging, and verifying credentials easier and more secure on the web.</p>
 
+      <p class="motivation">
+Readers that are new to this work should examine the <a href="https://www.w3.org/TR/vc-data-model/">latest Verifiable
```

Comment:
Is "examine" really the right word? Maybe "consult"?

But this is coming from a non-native English speaker, so feel free to ignore.